api/src/org/pentaho/platform/api/repository2/unified/RepositoryFile.java

```diff
@@ -192,12 +202,14 @@ public RepositoryFile( Serializable id, String name, boolean folder, boolean hid
   /*
    * This assumes all Serializables are immutable (because they are not defensively copied).
    */
-  public RepositoryFile( Serializable id, String name, boolean folder, boolean hidden, boolean versioned,
+  public RepositoryFile( Serializable id, String name, boolean folder, boolean hidden, boolean schedulable,
```

Comment:
Why not create a new constructor that takes all of these parameters and leave the old one and let it call the new one with a default value for schedulable? then deprecate the previous version of the constructor if you don't want anyone to use it.